src/stream/src/executor/actor.rs

```diff
@@ -56,6 +75,23 @@ impl ActorContext {
             .or_default()
             .push(err);
     }
+
+    pub fn store_mem_usage(&self, val: usize) {
+        // Record the last mem val.
+        // Calculate the difference between old val and new value, and apply the diff to total
+        // memory usage value.
+        let old_value = self.cur_mem_val.load(Ordering::Relaxed);
+        self.last_mem_val.store(old_value, Ordering::Relaxed);
+        let diff = val as i64 - old_value as i64;
+
+        self.total_mem_val.inc(diff);
+
+        self.cur_mem_val.store(val, Ordering::Relaxed);
+    }
+
+    pub fn get_mem_usage(&self) -> usize {
```

Comment:
just `mem_usage` in use naming convention
https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter